plugins/defaultplugins/l2plugin/vppcalls/compat_vppcalls.go

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) 2018 Cisco and/or its affiliates.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package vppcalls
+
+import (
+	govppapi "git.fd.io/govpp.git/api"
+	"github.com/ligato/cn-infra/logging"
+	"github.com/ligato/vpp-agent/plugins/defaultplugins/common/bin_api/l2"
+)
+
+func CheckMsgCompatibilityForXConnect(log logging.Logger, vppChan VPPChannel) error {
```

Comment:
Why complicate this with another function if all it does is put list of types to the function? It was much simpler before and better accessible as a exported variable.

Reply:
ok, I'll revert it. I just wanted to keep checking consistent across the plugins (the same principle is used in the ifconfig)

Reply:
It'd probably be better if we get rid of this `CheckMsgCompatibility...` functions since they don't really give any value, quite the contrary, they hide the list of requests inside function and even slightly decrease coverage, thus create burden to add useless tests.

Reply:
agree